scripts/build-avalon-image.sh

```diff
@@ -196,15 +197,25 @@ build_cgminer() {
         eval AVA_TARGET_PLATFORM=\${"`echo ${AVA_TARGET_BOARD//-/_}`"_brdcfg[0]}
         cd ..
         mkdir -p ./bin/${AVA_TARGET_BOARD}
-        cp ./openwrt/bin/${AVA_TARGET_PLATFORM}/packages/cgminer/cgminer*.ipk  ./bin/${AVA_TARGET_BOARD}
+        if [ "$AVA_MACHINE" == "avalon8" ]; then
```

Comment:
Yes. I try to use a commom structure for every machine, but some avalon7's machine can not build successfully, because some hardware will not support, eg. zctrl. Or if you have another  idea.
Thanks.